lib/widgets/molecules/multi_select_chips.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:flutter/material.dart';
+
+//class chip
+class Chip {
+  final String chipName;
+  final IconData? icon;
+
+  const Chip({
+    required this.chipName,
+    this.icon,
+  });
+}
+
+//example
+CreateMultiSelectChips createMultiSelectChipsExample() {
+  return const CreateMultiSelectChips(
+    chips: [
+      Chip(chipName: 'Weekly check-ins'),
+      Chip(chipName: 'Monthly check-ins'),
+      Chip(chipName: 'One-off sessions'),
+      Chip(chipName: 'Informal chats'),
+      Chip(chipName: 'Formal meetings'),
+      Chip(chipName: 'Long term'),
+    ],
+  );
+}
+
+//example 2
+CreateMultiSelectChips createMultiSelectChipsExampleWithIcon() {
+  return const CreateMultiSelectChips(
+    chips: [
+      Chip(chipName: 'Administrative Services', icon: Icons.campaign_outlined),
+      Chip(chipName: 'Agriculture & Forestry', icon: Icons.campaign_outlined),
+      Chip(
+          chipName: 'Architecture & Engineering',
+          icon: Icons.campaign_outlined),
+      Chip(
+          chipName: 'Arts, Entertainment, & Recreation',
+          icon: Icons.campaign_outlined),
+      Chip(
+          chipName: 'Beauty, Hair, & Cosmetics', icon: Icons.campaign_outlined),
+      Chip(
+          chipName: 'Building & Grounds Maintenance',
+          icon: Icons.campaign_outlined),
+    ],
+    maxSelection: 3,
+  );
+}
+
+//build
+class CreateMultiSelectChips extends StatelessWidget {
+  final List<Chip> chips;
+  final int? maxSelection;
+  const CreateMultiSelectChips({
+    Key? key,
+    required this.chips,
+    this.maxSelection,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    String textString = maxSelection != null
+        ? "Select up to $maxSelection (optional)"
+        : "Select all that apply (optional)";
+
+    return Column(
+      children: [
+        Center(
+          child: Text(
+            textString,
+            style: theme.textTheme.bodyMedium?.copyWith(
+              color: theme.colorScheme.secondary,
+            ),
+          ),
+        ),
+        const SizedBox(
+          height: 20,
+        ),
```

Comment:
This is essentially being used as padding, so you could use one of the predefined Insets.padding constants.